http/jaxrs/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
This file seems redundant. And there are others (dockerfiles etc.) which may be left out. Do we want to keep them? WDYT?

Reply:
Is redundant!